src/Directory.Build.props

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.3" />
+    <PackageReference Include="Neo" Version="3.0.3-CI01319" />
```

Comment:
move to 3.1.0

Reply:
Wait for https://github.com/neo-project/neo-modules/pull/673

Reply:
> 
Merged.